pkg/project/tfvc.go

```diff
@@ -0,0 +1,74 @@
+package project
+
+import (
+	"fmt"
+	"path/filepath"
+	"runtime"
+
+	"github.com/wakatime/wakatime-cli/pkg/log"
+
+	"github.com/yookoala/realpath"
+)
+
+// Tfvc contains tfvc data.
+type Tfvc struct {
+	// Filepath contains the entity path.
+	Filepath string
+}
+
+// Detect gets information about the tfvc project for a given file.
+func (t Tfvc) Detect() (Result, bool, error) {
+	fp, err := realpath.Realpath(t.Filepath)
+	if err != nil {
+		return Result{}, false,
+			Err(fmt.Errorf("failed to get the real path: %w", err).Error())
+	}
+
+	// Take only the directory
+	if fileExists(fp) {
+		fp = filepath.Dir(fp)
+	}
+
+	tfFolderName := ".tf"
+	if runtime.GOOS == "windows" {
```

Comment:
You skip the test for windows, and here you're added windows specific behavior. Did not fully understand, why you skipped testing for windows. Could you elaborate?